less/home-page.less

```diff
@@ -1,30 +1,88 @@
 // Home Content
 
-.home {
+#main-title {
+  display: flex;
+  width: 100%;
+  justify-content: center;
+  margin: 1.125% 0 3% 0;
```

Comment:
if you have percentage based margins or paddings, that usually causes your nav text to go on another line.  here is the styling code to keep the nav text on the same line
![image](https://user-images.githubusercontent.com/30554629/60305684-09360e80-9903-11e9-970a-3a29c3525dc9.png)